### DISCUSSION TITLE
Problem with upgrading to 11.3.0

### DISCUSSION THREAD

**Discussion by michael** [vitaly-t/pg-promise#869] [General]
Hi there!

Wanted to upgrade to 11.3.0 but it seems that some internals have changed, which breaks the code I put in place to have column names and JSON results automatically camelCased.

Here's a gist with my setup code, maybe you can give me pointers what changes I need to make to make it work again. :)

https://gist.github.com/michael/13892289de6a11071fec4616ac27689f

Thank you for your efforts writing and maintaining this library. I use it heavily every day! :)

**Comment by vitaly-t** [comment]
Those breaking changes were documented within [v11 release](https://github.com/vitaly-t/pg-promise/releases/tag/11.0.0).

And in case of [receive](https://vitaly-t.github.io/pg-promise/global.html#event:receive), here's the change for your code:

```ts
const pgOptions = {
    receive({ data }) {
        camelizeColumns(data);
    }
};
```

**Reply by michael** [reply]
Forgot to say thanks. Works perfectly!
